processes/runSetup.R

```diff
@@ -168,7 +168,20 @@ if (platform == 'Linux'){
   tempwd <- getwd()
   rundir <- paste(tempwd, "/assessment/ASAP/Run", '_', rand, sep = "")
   dir.create(path = rundir)
-  from.path <- paste('../EXE/ASAP3.EXE', sep = "")
+  
+  # setup command to run ASAP. For now, ASAP is located at /net/home2/mlee/admb-12.3/asap3/asap3
+  if(runClass=='neptune'){
```

Comment:
Yes, we can look into the best place to set this. For long-term we may not want runClass=='neptune'

Reply:
I agree.  Once native ASAP is tested on the HPCC servers, we can get rid of this.  I can't test it though.

I think what we could do is require the compiled linux ASAP3 executable to be put into a particular place.  This could be in inside the project, like in a folder called "external" or something.

Reply:
Great, thanks. We are going to work on testing on this end.